openlibrary/templates/type/edition/view.html

```diff
@@ -441,12 +441,11 @@ <h3>$_("The Physical Object")</h3>
             <h3>$_("Edition Identifiers")</h3>
             <dl class="meta">
                 $:display_identifiers("Open Library", [storage(url=None, value=edition.key.split("/")[-1])])
-                $ no_index = edition.get_ia_meta_fields().get('noindex', False)
```

Comment:
Do we still want this check?

Reply:
I think if we can avoid a call to metadata, that's better...